test/tariff_prep/test_tariff_prep.py

```diff
@@ -213,8 +213,11 @@ def test_mask_ranks(prep, censored, scores, ranks, expected):
     uniform_list_pos = 999
     min_cause_score = defaultdict(lambda: 0)
 
-    prep.mask_ranks([va], uniform, cutoffs, demog_restrictions, lowest_rank,
-                    uniform_list_pos, min_cause_score)
+    prep.mask_ranks([va],
+                    len(uniform), # this seems better matched to the doc string than what was here before converting to py3
+                    cutoffs, demog_restrictions, lowest_rank,
+                    uniform_list_pos/lowest_rank,  # I picked this because the doc string says it should be a value between 0 and 1, but I'm not sure it is correct
```

Comment:
Yeah, oof this test seems wonky period. Not sure if this should be `/lowest_rank` or `/len(uniform)`, or why it wasn't before. My north star to follow here is whatever intellectually we know the test _should_ be testing.